.github/workflows/build.yaml

```diff
@@ -59,6 +59,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     timeout-minutes: 120
+    if: inputs.os != 'windows' || (!startsWith(inputs.version, '1.3') && inputs.version != 'main')
```

Comment:
Maybe better to skip this in `build-and-release`?

Reply:
doesn't work :/